ci-operator/step-registry/openshift/e2e/test/openshift-e2e-test-commands.sh

```diff
@@ -128,6 +128,11 @@ openstack*)
     export TEST_PROVIDER='{"type":"openstack"}';;
 ovirt) export TEST_PROVIDER='{"type":"ovirt"}';;
 kubevirt) export TEST_PROVIDER='{"type":"kubevirt"}';;
+ibmcloud)
+    export TEST_PROVIDER=ibmcloud
```

Comment:
I think I'm going to have to drop this `TEST_PROVIDER`, since I think the `openshift-tests` support for IBM Cloud will take some time to work through. I'm hoping we can get away with the `skeleton` or another provider instead for the time being, until we determine how many changes are required to support the `ibmcloud` provider.

Reply:
Swapped to `skeleton` for now (as a 'default' provider)